worlds/hk/__init__.py

```diff
@@ -216,7 +218,10 @@ def fill_slot_data(self):
         options = slot_data["options"] = {}
         for option_name in self.options:
             option = getattr(self.world, option_name)[self.player]
-            options[option_name] = int(option.value)
+            if isinstance(option.value, (list, dict)):
```

Comment:
Why check for (list, dict) at all?

Reply:
well, we can't cast those to int and if I change the dict from strict {str: int} mapping I expect to be shouted at, so I figured just drop the data.

Reply:
I'm not asking why you're dropping it, I'm asking why you're specifically checking for those instead of just 'if isinstance(option.value, int):'

Reply:
well, I can do that too, but that'd be (int, bool)

Reply:
Do whatever works but isn't unnecessary. The check for (list,dict) seemed unnecessary.

Reply:
now making use of https://github.com/python/cpython/issues/84403 to skip unnecessary checking

Reply:
Exception handling for control flow is smelly to me, but since it's 'cost effective' I suppose it's okay.